docs/BR.md

```diff
@@ -1532,6 +1651,11 @@ The CA SHALL record at least the following events:
    4. Issuance of Certificates; 
    5. Generation of Certificate Revocation Lists; and 
    6. Signing of OCSP Responses (as described in [Section 4.9](#49-certificate-revocation-and-suspension) and [Section 4.10](#410-certificate-status-services)).
+   7. Multi-Perspective Issuance Corroboration attempts from each Network Perspective, minimally recording the following information:
+      - an identifier that uniquely identifies the perspective used
+      - the attempted domain name
```

Comment:
MPIC is required for some IP Address Validation methods, so this needs to be something more general than "domain name".

Reply:
[Updated](https://github.com/ryancdickson/staging/pull/8/commits/8e5189533480cd185e17670f806c3d79c0753da0) bullets 7 and 8 to include consideration for IP addresses.